Formula/r/reprepro.rb

```diff
@@ -36,16 +36,20 @@ class Reprepro < Formula
     cause "No support for GNU C nested functions"
   end
 
+  # Replace strcmp2 with strcmp for consistent string comparison
+  patch do
+    url "https://raw.githubusercontent.com/Homebrew/formula-patches/5426d605c0d6b6bf4c2e315fed7a991f447caf1a/reprepro/5.4.4-strcmp2.patch"
+    sha256 "cf1aacf08960cb89896a739086d170417820fe894044d0f446b4ff303a1182e5"
+  end
```

Comment:
Has this been upstreamed?

Reply:
not really, I can push it now.

Reply:
https://salsa.debian.org/debian/reprepro/-/merge_requests/10